src/Tribe/Views/V2/Hooks.php

```diff
@@ -147,6 +148,25 @@ public function action_disable_assets_v1() {
 		$assets->disable_v1();
 	}
 
+	public function action_register_widget_styles() {
+		$style_dir = plugin_dir_url( TRIBE_EVENTS_FILE ) . 'src/resources/css/';
+
+		wp_register_style(
+			'tribe-events-widgets-v2-skeleton',
+			$style_dir . 'widgets-skeleton.css',
+			[ 'tribe-common-skeleton-style' ],
+		);
+
+		wp_register_style(
```

Comment:
:no_entry_sign: Resource version not set in call to wp_register_style(). This means new versions of the style will not always be loaded due to browser caching.

<sub>hash: bd355879d3de32e83b98c5dd340cfd0b</sub>